### PR TITLE
CU-8695kqkkc_Delete-minyan

### PR DESCRIPTION
**Pull request by SaraCohenNBS**

### CODE REVIEW

**Comment by benshabatnoam** [comment]
Task linked: [CU-8695kqkkc Delete minyan](https://app.clickup.com/t/8695kqkkc)

File: client/src/pages/minyanim/components/zmanim-table.tsx [hunk 1]
```diff
@@ -17,17 +17,18 @@ import dayjs, { Dayjs } from 'dayjs';
 import { response } from 'express';
```
Comment (SaraCohenNBS) [inline client/src/pages/minyanim/components/zmanim-table.tsx:17]: Remove unused imports

File: client/src/pages/minyanim/components/zmanim-table.tsx [hunk 2]
```diff
@@ -17,17 +17,18 @@ import dayjs, { Dayjs } from 'dayjs';
...
 import type { ColumnDef } from '@/components/core/data-table';
+import zIndex from '@mui/material/styles/zIndex';
 
 const styleTypography = {
   display: 'grid',
   justifyItems: 'center',
   alignItems: 'center',
   whiteSpace: 'nowrap',
-  height: '54px',
+  height: '54px'
```
Comment (SaraCohenNBS) [inline client/src/pages/minyanim/components/zmanim-table.tsx:31]: missing format

File: client/src/pages/minyanim/components/zmanim-table.tsx [hunk 3]
```diff
@@ -163,24 +171,26 @@ export function ZmanimTable(props: { typeDate: string }): React.JSX.Element {
       tooltip: 'Time to start Blink before lights on',
     },
     {
```
Comment (SaraCohenNBS) [inline client/src/pages/minyanim/components/zmanim-table.tsx:173]: The columns are a constant variable that does not depend on changes that occur in the component, so we will define it outside the component
It is possible to set it on this page before the component and it is also possible to set it on the config page

**Review by SaraCohenNBS — COMMENTED** [review]
I added some comments for improvement, any changes following them will be in a new PR and a new branch that came out of main

**Pull request merged**